apps/web/src/common/modules/navigations/gnb/modules/gnb-recent-favorite/modules/GNBFavorite.vue

```diff
@@ -247,10 +321,40 @@ export default {
                         name: itemInfo[2],
                     },
                 }).catch(() => {});
+            } else if (item.itemType === SUGGESTION_TYPE.COST_ANALYSIS) {
+                const dataSourceId = state.favoriteCostAnalysisItems.find((d) => d.name === itemName)?.dataSourceId;
+                SpaceRouter.router.push({
+                    name: COST_EXPLORER_ROUTE.COST_ANALYSIS.QUERY_SET._NAME,
+                    params: {
+                        dataSourceId,
+                        costQuerySetId: itemName,
+                    },
+                }).catch(() => {});
             }
             emit('close');
         };
 
+        const costQuerySetFetcher = getCancellableFetcher(SpaceConnector.clientV2.costAnalysis.costQuerySet.list);
+
+        const getAllCostQuerySetList = async () => {
+            try {
+                const { status, response } = await costQuerySetFetcher({
+                    query: {
+                        filter: [{ k: 'user_id', v: store.state.user.userId, o: 'eq' }],
+                        only: ['cost_query_set_id', 'data_source_id', 'name'],
+                    },
+                });
+                if (status === 'succeed' && response?.results) {
+                    state.costQuerySets = response.results;
+                } else {
+                    state.costQuerySets = [];
```

Comment:
suggestion: else case is cancelled case. I think UX is very poor if set empty array when api call is cancelled due to the next api call. 
I think it would be better just doing nothing on else case.